source/release_notes.rst

```diff
@@ -7,19 +7,22 @@ ww201818
 ========
 
 ================================  =====
-Build ID                          celadon-userdebug 8.1.0 OPM3.171019.013 eng.build.20180502.092142 test-keys
+Build ID                          celadon-userdebug 8.1.0 O.CEL.T.WW24 eng.build.20180618.145425 test-keys
 Software Version                  Android version 8.1.0
 Mesa                              18.2.0-devel
 Kernel Flinger                    5.09
 Kernel Version / Android Version  4.14.35-project-celedon_64-g0a945a1
-Download Link                     https://github.com/projectceladon/celadon-binary/tree/master/ww201818
+Download Link                     https://github.com/projectceladon/celadon-binary/tree/master/ww201824
```

Comment:
You will need to create a `ww201824` folder and host the prebuilt binary there, otherwise this is a invalid link.